### PR TITLE
missing file

### PR DESCRIPTION
**Pull request by m0n4**
Hi,

thank you for this tool.
I added the missing file and improved a thing or two.

greetings

### CODE REVIEW

File: pkt-mining-stats.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-killall -9 python3
+pkill -f get_address_stats.py
+pkill -f get_pool_stats.py
+pkill -f pkt-mining-stats.py
```
Comment (empewoow) [inline pkt-mining-stats.sh:5]: This is better indeed, glad you noticed it. It was really quick and dirty.

File: settings.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+myHost = '127.0.0.1'
+myPort = 8050
+timeScale = 1 # in days
```
Comment (empewoow) [inline settings.py]: Good idea! Make it default to 2? That's what it says in the docs now...

**Review by empewoow — CHANGES_REQUESTED** [review]
Hey m0n4,

Many thanks for your contribution! Can you change that one variable in `settings.py` from 1 day to 2 days by default, then I'll merge your edits.

You can also contact me on Telegram and pkt.chat. You can find me in these chats with my username. Are you there as well?

Regards,

Matthijs

**Review by empewoow — APPROVED** [review]
Awesome!

File: settings.py (hunk 2 above)
Comment (m0n4) [reply settings.py]: yes of course !

**Pull request merged**